### PR TITLE
Feature/back button

### LINKED ISSUE #9: As a user I should be able to go back to the home page from a movies description

### PR DESCRIPTION
**Pull request by pareesakd1118**
## Describe your changes
Add functionality to back button so takes you from single movie description to main page and add error handling. 
## Issue ticket number if applicable 
This pull request closes #9

## Checklist before requesting a review
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.

## Next Steps
Continue styling and move to next iterations.

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -9,17 +9,20 @@ function App() {
   const [movies, setMovies] = useState([]);
   const [selectedMovie, setSelectedMovie] = useState({});
   const [isMovieSelected, setIsMovieSelected] = useState(false);
+  const [isError, setIsError] = useState(false)
+  const [responseLevel, setResponseLevel] = useState(undefined)
```
Comment (kaylagordon) [inline src/App/App.js:13]: Typically we use `null` to set a "nothing" value as default. `undefined` isn't something we purposely set a value to.

File: src/App/App.js [hunk 2]
```diff
@@ -30,45 +33,46 @@ function App() {
...
     <React.Fragment>
-      <Nav />
-      <React.Fragment>
-        {!isMovieSelected ? (
-          <Main showDesc={showDesc} movieData={movies} />
-        ) : (
-          <Desc movie={selectedMovie} />
-        )}
-      </React.Fragment>
+      {isError ? (<p className="error">ERROR: {responseLevel} </p>) : (
```
Comment (kaylagordon) [inline src/App/App.js:62]: I like the way you made this dynamic to show the error code. I might consider adding more messaging for the user. As a user, what should I do if I get an error? Return to the home page? Try again? Leave and come back later?

File: src/App/App.js [hunk 3]
```diff
@@ -30,45 +33,46 @@ function App() {
...
     setIsMovieSelected(true);
   }
+
+  function displayHomePage() {
+    setIsMovieSelected(false)
+    setSelectedMovie({})
+  }
+
+  function displayError(responseLevel) {
+    setIsError(true)
```
Comment (kaylagordon) [inline src/App/App.js:56]: Do we need `isError` to exist? Can we just use `responseLevel` to know if there is an error? i.e. if `responseLevel` is `null`, we know there is no error.

File: src/App/App.js [hunk 4]
```diff
@@ -30,45 +33,46 @@ function App() {
...
+      {isError ? (<p className="error">ERROR: {responseLevel} </p>) : (
+        <React.Fragment>
+          <Nav />
+          <React.Fragment>
+          {!isMovieSelected ? (
+            <Main showDesc={showDesc} movieData={movies} />
+            ) : (
+            <Desc movie={selectedMovie} displayHomePage={displayHomePage}/>
+          )}
+          </React.Fragment>
```
Comment (kaylagordon) [inline src/App/App.js:71]: `React.Fragment` seems to be deprecated ([link here](https://legacy.reactjs.org/docs/fragments.html)) and it's now recommended to use [this syntax](https://react.dev/reference/react/Fragment). Also, typically you only use fragments to surround the entire JSX in one element. You don't typically see them nested in the way you have them. You want to use typicaly HTML elements where you can.

**Review by EthanDuvall — APPROVED** [review]
Fixing comments

**Pull request merged**